hibernate-core/src/main/java/org/hibernate/boot/model/source/spi/CollectionIdSource.java

```diff
@@ -30,4 +32,9 @@
 	 * @return The identifier value generator name
 	 */
 	public String getGeneratorName();
+
```

Comment:
Hi @gbadner ,
if the application implements the CollectionIdSource yes, but looking at the use we do, not able to figure out why an app should do this.